gammapy/estimators/utils.py

```diff
@@ -770,83 +773,350 @@ def combine_flux_maps(maps, method="gaussian_errors", reference_model=None):
         * gaussian_errors :
             Under the gaussian error approximation the likelihood is given by the gaussian distibution.
             The product of gaussians is also a gaussian so can derive dnde, dnde_err, and ts.
+        * distrib :
+            Likelihood profile approximation assuming that probabilities distributions for
+            flux points correspond to asymmetric gaussians and for upper limits to complementary error functions.
+            Use available quantities among dnde, dnde_err, dnde_errp, dnde_errn, dnde_ul, and ts.
+        * profile :
+            Sum the likelihood profile maps.
+            The flux maps must contains the `stat_scan` maps.
+
+        Default is "gaussian_errors" which is the faster but least accurate solution,
+        "distrib"  will be more accurate if dnde_errp and dnde_errn are available,
+        "profile"  will be even more accurate if "stat_scan" is available.
+
     reference_model : `~gammapy.modeling.models.SkyModel`, optional
         Reference model to use for conversions.
-        If None, use the reference_model of the first FluxMaps in the list.
+        Default is None and is will use the reference_model of the first FluxMaps in the list.
+
+    dnde_scan_axis : `~gammapy.maps.MapAxis`
+        dnde axis used to compute the likelihood profile.
```

Comment:
When reading this docstring, I should admit that this is hard to understand what is such dnde axis... maybe adding an example could help

Reply:
I will add a tutorial after that all the new TSmap related features are merged.